card/mod/05_standard/spec/set/right/password_spec.rb

```diff
@@ -31,15 +31,15 @@
         pw_card.save
         expect(original_pw).to eq(pw_card.refresh(force=true).content)
       end
-      
+
       it "shouldn't break email editing" do
-        @user_card.account.update_attributes! :subcards=>{'+*password'=>'', '+*email'=>'joe2@user.com'}
+        @user_card.account.update_attributes! subcards: {'+*password'=>'', '+*email'=>'joe2@user.com'}
```

Comment:
Line is too long. [102/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.